tests/ui_tests/test_adcm_upgrade.py

```diff
@@ -0,0 +1,87 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import threading
+from typing import Tuple
+
+import pytest
+import allure
+
+from selenium.common.exceptions import StaleElementReferenceException
+from adcm_client.objects import ADCMClient
+from adcm_pytest_plugin.plugin import parametrized_by_adcm_version
+from adcm_pytest_plugin.utils import wait_until_step_succeeds
+
+from tests.upgrade_utils import upgrade_adcm_version
+from tests.ui_tests.app.app import ADCMTest
+from tests.ui_tests.app.pages import BasePage
+from tests.ui_tests.app.page.admin.page import AdminIntroPage
+from tests.ui_tests.app.page.login.page import LoginPage
+from tests.ui_tests.app.page.cluster_list.page import ClusterListPage
+from tests.ui_tests.app.page.bundle_list.page import BundleListPage
+from tests.ui_tests.app.page.profile.page import ProfilePage
+
+# pylint: disable=redefined-outer-name
+
+
+def old_adcm_image():
+    return parametrized_by_adcm_version(adcm_min_version="2021.03.10")[0][-1]
```

Comment:
is it correct that in tests/functional/test_adcm_upgrade.py use
`return parametrized_by_adcm_version(adcm_min_version="2019.10.08")[0]` min version and without second index?

Reply:
Yes, it's correct, because in functional tests we check upgrades from many versions, but we need just values for parametrization.
Here we want to check the most recent version, because of locators etc. etc.